libcxx/docs/Status/Cxx23Papers.csv

```diff
@@ -61,7 +61,7 @@
 "`P1899R3 <https://wg21.link/P1899R3>`__","``stride_view``","2022-07 (Virtual)","","",""
 "`P2093R14 <https://wg21.link/P2093R14>`__","Formatted output","2022-07 (Virtual)","|Complete|","18.0",""
 "`P2165R4 <https://wg21.link/P2165R4>`__","Compatibility between ``tuple``, ``pair`` and ``tuple-like`` objects","2022-07 (Virtual)","","",""
-"`P2278R4 <https://wg21.link/P2278R4>`__","``cbegin`` should always return a constant iterator","2022-07 (Virtual)","","",""
+"`P2278R4 <https://wg21.link/P2278R4>`__","``cbegin`` should always return a constant iterator","2022-07 (Virtual)","|Complete|","20.0","This paper is applied as DR against C++23 (MSVC STL and libstdc++ have done the same). The feature macro is not yet defined, since other features from the original paper have not been completed."
```

Comment:
> Also, since this is a C++23 paper, it seems wrong to say that the paper is being applied as a DR against C++23. It's just a C++23 paper implemented in C++23 (and above). Or did I miss something?

Looks like that something was mistakenly duplicated in force-pushing. It was right in 160a204f24092aeffcb9674ce4c3093949301535.

```suggestion
"`P2278R4 <https://wg21.link/P2278R4>`__","``cbegin`` should always return a constant iterator","2022-07 (Virtual)","|In Progress|","20.0","``basic_const_iterator`` has been implemented, and the const range accessors (``cbegin``, ``cend``, ``crbegin``, ``crend``) have been updated to return this new type. However, the changes to ``cdata`` and ``span``, and ``const_view``, have not yet been implemented."
```

Reply:
As to your first question, originally the plan was to implement this in two parts for ease of review; however, since I've been waiting for review for so long, I did end up implementing the other parts of the paper. I can certainly merge that PR into this one.

On the second point... I need to look through this and figure out exactly what's going on, it's been a while since I've looked.

Reply:
Second point: I think this was an incorrect rebase; this note was intended to apply to a different paper (P2836R1).

Reply:
I'm going to do a re-rebase I think, since I see other things that were not merged correctly - sorry.